collection-scripts/gather_trigger_gmr

```diff
@@ -45,6 +48,56 @@ cinder_trigger_gmr() {
 }
 
 
+nova_trigger_gmr() {
+    # https://docs.openstack.org/nova/latest/reference/gmr.html
+    echo "Trigger GMR for Nova services"
+
+    schedulers=`$oc get pod -l service=nova-scheduler -o=custom-columns=N:.metadata.name --no-headers | tr '\n' ' '`
+    for scheduler in $schedulers
+    do
+        run_bg $oce $scheduler -- pkill --signal USR2 nova-scheduler
+    done
+
+    conductors=`$oc get pod -l service=nova-conductor -o=custom-columns=N:.metadata.name --no-headers | tr '\n' ' '`
```

Comment:
We are trying to do as few requests as possible to reduce the time it takes to run the must-gather, because GSS has complained about how long it takes to run.  For this reason it would be best to do just 1 request like we do in the cinder service.

Reply:
let me see if I can grab all the pods in a single request

Reply:
I reduced the oc requests